File: chatgpt-generated/python/xss.py
from flask import Flask, request

app = Flask(__name__)

@app.route('/comment')
def comment():
    user_comment = request.args.get('comment', '')
    return f"<h2>User Comment:</h2><p>{user_comment}</p>"

if __name__ == '__main__':
    app.run(debug=True)

File: chatgpt-generated/go/xss.go
package main

import (
	"fmt"
	"net/http"
)

func main() {
	http.HandleFunc("/xss", func(w http.ResponseWriter, r *http.Request) {
		comment := r.URL.Query().Get("comment")
		fmt.Fprintf(w, "<h2>User Comment:</h2><p>%s</p>", comment)
	})
	http.ListenAndServe(":8080", nil)
}
